pkg/cmd/objects/delete/delete.go

```diff
@@ -39,22 +43,37 @@ func NewDeleteCmd(f *cmdutil.Factory, runF func(*DeleteOptions) error) *cobra.Co
 	}
 
 	cmd := &cobra.Command{
-		Use:               "delete <index> --object-ids <object-ids> --confirm",
+		Use:               "delete <index> [--object-ids <object-ids> | <filters>...] [--confirm] [--wait]",
 		Args:              validators.ExactArgs(1),
 		ValidArgsFunction: cmdutil.IndexNames(opts.SearchClient),
 		Short:             "Delete objects from an index",
 		Long: heredoc.Doc(`
 			This command deletes the objects from the specified index.
+
+			You can either directly specify the objects to delete by theirs IDs or use the filters related flags to delete the matching objects.
```

Comment:
may sound a little Nit-picky: but since they are not mutually exclusive...

```suggestion
			You can either directly specify the objects to delete by theirs IDs and/or use the filters related flags to delete the matching objects.
```

Reply:
👍